src/material-experimental/mdc-form-field/_form-field-native-select.scss

```diff
@@ -92,20 +93,62 @@ $mat-form-field-select-horizontal-end-padding: $mat-form-field-select-arrow-widt
   }
 }
 
+// Gets the color to use for some text that is highlighted while a select has focus.
+@function _get-focused-arrow-color($palette) {
+  @return rgba(mdc-theme-color.prop-value($palette), 0.87);
+}
+
 @mixin private-form-field-native-select-color($config) {
-  select.mat-mdc-input-element {
-    // On dark themes we set the native `select` color to some shade of white,
-    // however the color propagates to all of the `option` elements, which are
-    // always on a white background inside the dropdown, causing them to blend in.
-    // Since we can't change background of the dropdown, we need to explicitly
-    // reset the color of the options to something dark.
-    @if (map.get($config, is-dark)) {
-      option {
-        color: palette.$dark-primary-text;
+  @include mdc-helpers.mat-using-mdc-theme($config) {
+    $_arrow-color: rgba(mdc-theme-color.prop-value(on-surface), 0.54);
```

Comment:
You don't need the underscore prefix for these variables, they're scoped to the mixin.